cli/wsproxy.ml

```diff
@@ -28,7 +28,7 @@ let start path handler =
   Lwt.catch
     (fun () -> Lwt_unix.unlink fd_sock_path)
     (fun _ -> Lwt.return_unit) >>= fun () ->
-  let () = Lwt_unix.bind fd_sock (Unix.ADDR_UNIX fd_sock_path) in
+  Lwt_unix.bind fd_sock (Unix.ADDR_UNIX fd_sock_path) >>= fun () ->
```

Comment:
Would this leak the socket if the bind fails? It would seem it does, but then previously it would've leaked it as well.

Reply:
Yes but that would fail the whole thing and exit from `wsproxy`